app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+module Idv
+  class AamvaStateMaintenanceWindow
+    MAINTENANCE_WINDOWS = {
+      'CA' => [
+        # Daily, 4:00 - 5:30 am. ET.
+        { cron: '0 4 * * *', duration: 90 },
+        # Monday, 1:00 - 1:45 am. ET
+        { cron: '0 1 * * Mon', duration: 45 },
+        # Monday, 1:00 - 4:30 am. ET on 1st and 3rd Monday of month.
+        { cron: '0 1 * * Mon#1', duration: 3.5 * 60 },
+        { cron: '0 1 * * Mon#3', duration: 3.5 * 60 },
+      ],
+      'CT' => [
+        # Daily, 4:00 am. to 6:30 am. ET.
+        { cron: '0 4 * * *', duration: 90 },
+        # Sunday 6:00 am. to 9:30 am. ET
+        { cron: '0 6 * * Mon', duration: 3.5 * 60 },
+      ],
+      'DC' => [
+        # Daily, Midnight to 6 am. ET.
+        { cron: '0 0 * * *', duration: 6 * 60 },
+      ],
+      'DE' => [
+        # Daily, Midnight to 5 am. ET.
+        { cron: '0 0 * * *', duration: 5 * 60 },
+      ],
+      'FL' => [
+        # Sunday 7:00 am. to 12:00 pm. ET
+        { cron: '0 7 * * Sun', duration: 5 * 60 },
+      ],
+      'IA' => [
+        # "Daily system resets, normally at 4:45 am. to 5:15 am ET."
+        { cron: '45 4 * * *', duration: 30 },
+      ],
+      'IN' => [
+        # Sunday morning maintenance from 6 am. to 10 am. ET.
+        { cron: '0 6 * * Sun', duration: 4 * 60 },
+      ],
+      'IL' => [
+        { cron: '30 2 * * *', duration: 2.5 * 60 }, # Daily, 2:30 am. to 5 am. ET.
+      ],
+      'KY' => [
+        # Daily maintenance from 2:50 am. to 6:40 am. ET
+        { cron: '50 2 * * *', duration: 230 },
```

Comment:
Would `duration: (3 * 60) + 50` be too weird? I tried to keep these intuitive (e.g., `3.5 * 60` for 3.5 hours), but this is the outlier.